### PR TITLE
Update where to check for existing chapters

### PR DESCRIPTION
**Pull request by MounaBelaid**

### CODE REVIEW

File: content/coordination/onboarding/index.en.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ weight: 1
 ## Goal
 
 The goal of this activity is to start a conversation with prospective
-organisers, answer any questions and send information on how to get a
+organizers, answer any questions and send information on how to get a
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:10]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

File: content/coordination/onboarding/index.en.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ timely.
     Team, the messages to this account are forwarded to the email
     address you provided during your own onboarding process.
 
--   On the rladies.org website, we ask any prospective organisers to get
+-   On the rladies.org website, we ask any prospective organizers to get
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:21]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

File: content/coordination/onboarding/index.en.md [hunk 3]
```diff
@@ -31,25 +31,25 @@ timely.
...
 
--   Search our database on GitHub
-    ([https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv](https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv))
-    to make sure the city does not have a chapter yet.
+-   Make a search on GitHub
+    ([https://github.com/rladies/rladies.github.io/tree/main/data/chapters](https://github.com/rladies/rladies.github.io/tree/main/data/chapters))
+    to make sure the city does not have a chapter yet. You can use CTRL+F to search using the city's name.
 
--   If there is a chapter already, check the name of the organisers
+-   If there is a chapter already, check the name of the organizers
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:38]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

File: content/coordination/onboarding/index.en.md [hunk 4]
```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
 
         -   Country
 
-        -   Organisers
+        -   Organizers
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:63]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

File: content/coordination/onboarding/index.en.md [hunk 5]
```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
...
 
         -   Status = Prospective
 
--   Invite organisers to the organisers slack.
+-   Invite organizers to the organizers slack.
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:67]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

File: content/coordination/onboarding/index.en.md [hunk 6]
```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
...
 
--   Send the prospective organiser an email with all the most important
+-   Send the prospective organizer an email with all the most important
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:69]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

File: content/coordination/onboarding/index.en.md [hunk 7]
```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
...
+    -   People may ask to be added to the organizer slack, please check
+        if their name is in the meetup list of co-organizers. If not,
         ask why.
 
-    -   Change list of organisers:\
-        Check and amend the list of organisers here:\
-        Email former organiser for confirmation (see [template B](#appendix-b)).
+    -   Change list of organizers :\
+        Check and amend the list of organizers here:\
+        Email the former organizer for confirmation (see [template B](#appendix-b)).
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:80]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

File: content/coordination/onboarding/index.en.md [hunk 8]
```diff
@@ -111,8 +111,8 @@ In the Slack, together with all R-Ladies chapters organizers worldwide,
 you will find the
 R-Ladies Global leadership team. Feel free to reach out to them through
 the channel #ask_the_leadership if you have questions that are not
-addressed in the Starter Kit, the Guidelines, or that other organizers
-cannot answer. More info about R-Ladies Global team here:
+addressed in the Starter Kit, or the Guidelines, or that other organizers
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:114]:
```suggestion
addressed in the Guidelines or that other organisers
```

Starter-Kit no longer really contains relevant information, so we shouldnt mention it.

File: content/coordination/onboarding/index.en.md [hunk 9]
```diff
@@ -111,8 +111,8 @@ In the Slack, together with all R-Ladies chapters organizers worldwide,
...
+cannot answer. More info about the R-Ladies Global team is here:
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:115]:
```suggestion
cannot answer. More info about the R-Ladies Global team is can be found on the R-Ladies website:
```

File: content/coordination/onboarding/index.en.md [hunk 10]
```diff
@@ -143,29 +143,29 @@ we can add them to our records."
 
 ## Appendix B
 
-> confirm list of organisers - email template
+> confirm list of organizers - email template
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:146]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

File: content/coordination/onboarding/index.en.md [hunk 11]
```diff
@@ -143,29 +143,29 @@ we can add them to our records."
...
-    from the list of organisers?
+    from the list of organizers?
 
     -   \<\<NAME1\>\>
 
     -   \<\<NAME2\>\>
 
-We assume former organisers provide all the necessary information to new
+We assume former organizers provide all the necessary information to new
 organisers before passing the baton :)
```
Comment (drmowinckels) [inline content/coordination/onboarding/index.en.md:169]: "organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

**Review by drmowinckels — CHANGES_REQUESTED** [review]

File: content/coordination/onboarding/index.en.md (hunk 1 above)
Comment (MounaBelaid) [reply content/coordination/onboarding/index.en.md:10]: Thanks for letting me know.

Comment (rivaquiroga) [reply content/coordination/onboarding/index.en.md:10]:
```suggestion
organisers, answer any questions and send information on how to get a
```
I'll reverse the spelling to UK so we can merge these changes an update the Guide

File: content/coordination/onboarding/index.en.md (hunk 3 above)
Comment (rivaquiroga) [reply content/coordination/onboarding/index.en.md:38]:
```suggestion
-   If there is a chapter already, check the name of the organisers
```

File: content/coordination/onboarding/index.en.md (hunk 4 above)
Comment (rivaquiroga) [reply content/coordination/onboarding/index.en.md:63]:
```suggestion
        -   Organisers
```

File: content/coordination/onboarding/index.en.md (hunk 5 above)
Comment (rivaquiroga) [reply content/coordination/onboarding/index.en.md:67]:
```suggestion
-   Invite organisers to the organisers slack.
```

File: content/coordination/onboarding/index.en.md (hunk 6 above)
Comment (rivaquiroga) [reply content/coordination/onboarding/index.en.md:69]:
```suggestion
-   Send the prospective organiser an email with all the most important
```

File: content/coordination/onboarding/index.en.md (hunk 10 above)
Comment (rivaquiroga) [reply content/coordination/onboarding/index.en.md:146]:
```suggestion
> confirm list of organisers - email template
```

File: content/coordination/onboarding/index.en.md [hunk 12]
```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
...
 
--   Send the prospective organiser an email with all the most important
+-   Send the prospective organizer an email with all the most important
     information and links (see [Appendix A](#appendix-a))
 
 -   **Other types of requests:**
 
-    -   People may ask to be added to the organiser slack, please check
-        if their name is in the meetup list of co-organisers. If not,
+    -   People may ask to be added to the organizer slack, please check
```
Comment (rivaquiroga) [inline content/coordination/onboarding/index.en.md:74]:
```suggestion
    -   People may ask to be added to the organiser slack, please check
```

File: content/coordination/onboarding/index.en.md [hunk 13]
```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
...
+        if their name is in the meetup list of co-organizers. If not,
```
Comment (rivaquiroga) [inline content/coordination/onboarding/index.en.md:75]:
```suggestion
        if their name is in the meetup list of co-organisers. If not,
```

File: content/coordination/onboarding/index.en.md [hunk 14]
```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
...
         ask why.
 
-    -   Change list of organisers:\
-        Check and amend the list of organisers here:\
-        Email former organiser for confirmation (see [template B](#appendix-b)).
+    -   Change list of organizers :\
```
Comment (rivaquiroga) [inline content/coordination/onboarding/index.en.md:78]:
```suggestion
    -   Change list of organisers :\
```

File: content/coordination/onboarding/index.en.md [hunk 15]
```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
...
+        Check and amend the list of organizers here:\
```
Comment (rivaquiroga) [inline content/coordination/onboarding/index.en.md:79]:
```suggestion
        Check and amend the list of organisers here:\
```

File: content/coordination/onboarding/index.en.md (hunk 7 above)
Comment (rivaquiroga) [inline content/coordination/onboarding/index.en.md:80]:
```suggestion
        Email the former organiser for confirmation (see [template B](#appendix-b)).
```

File: content/coordination/onboarding/index.en.md [hunk 16]
```diff
@@ -143,29 +143,29 @@ we can add them to our records."
...
-> confirm list of organisers - email template
+> confirm list of organizers - email template
 
 ```markdown
 Hi \<\<first name\>\>,
 
-Thanks for sending through the updated list of organisers. I sent the
-slack invites to the new organisers.
+Thanks for sending through the updated list of organizers. I sent the
+slack invites to the new organizers.
```
Comment (rivaquiroga) [inline content/coordination/onboarding/index.en.md:152]:
```suggestion
slack invites to the new organisers.
```

File: content/coordination/onboarding/index.en.md [hunk 17]
```diff
@@ -143,29 +143,29 @@ we can add them to our records."
...
 GitHub:
 https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv.
 
--   Could you please send name and surname of the new organisers so that
+-   Could you please send name and surname of the new organizers so that
     I can update the chapter list?
 
 -   Could you also please confirm whether I should remove the following
-    from the list of organisers?
+    from the list of organizers?
```
Comment (rivaquiroga) [inline content/coordination/onboarding/index.en.md:162]:
```suggestion
    from the list of organisers?
```

**Comment by rivaquiroga** [comment]
Hi, @MounaBelaid! I commited all Mo's suggestions. There is one more edit we need you to do before merging: on line 156 the current-chapters.csv is mentioned. We need to change that to https://github.com/rladies/rladies.github.io/tree/main/data/chapters.

**Comment by drmowinckels** [comment]
> Hi, @MounaBelaid! I commited all Mo's suggestions. There is one more edit we need you to do before merging: on line 156 the current-chapters.csv is mentioned. We need to change that to https://github.com/rladies/rladies.github.io/tree/main/data/chapters.

I altered it, @rivaquiroga :)

**Comment by shannonpileggi** [comment]
hey @drmowinckels! it looks like your reviewer status still says "requested changes". have they all been addressed / is this ready for merge?

**Review by drmowinckels — APPROVED** [review]

**Pull request merged**